README.md

```diff
@@ -71,6 +79,7 @@ yotiClient.getActivityDetails(token).then((activityDetails) => {
 	}
 })
 ```
+Example reference: [node/example/simple-login/index.js.](https://github.com/getyoti/node/blob/master/example/simple-login/index.js)
```

Comment:
As this will also be on NPM, I would suggest `[Example reference](https://github.com/getyoti/node/blob/master/example/simple-login/index.js)`